content/management/configuration-api/v3.2/index.mdx

```diff
@@ -2785,6 +2785,230 @@ curl -X POST \
 
 </Section>
 
+# Auto access rules
+
+Auto access rules allow to assign groups to customer session according to their tracking url and geolocation.  Those rules are
```

Comment:
**Auto access** allows you to assign groups to a thread that will be created when a customer starts a chat. Those groups are assigned based on the conditions defined to match the customer's tracking URL and geolocation.  Every time a customer calls **Login** or **Update Customer Page**, our backend checks if the tracking URL and geolocation match the pre-defined conditions. Given there's a match, the groups are assigned to a thread, but not until a chat starts and the thread is actually created. 

If a customer starts a chat via **Start Chat** or **Activate Chat** with the `chat.access` parameter, `chat.access` overrides the auto access assigned to the thread.

Reply:
This is my suggestion. I know the desc is not perfect, so feel free to comment. Together with @bkubiak we decided to refer to a thread's access rather than the access of a customer/customer session.